src/Microsoft.Extensions.Logging.Analyzers/Microsoft.Extensions.Logging.Analyzers.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Analyzer package for Microsoft.Extensions.Logging.</Description>
```

Comment:
To make this work as an analyzer and not a compilation reference, we'll need to change the default package layout. The library needs to end up in analyzers/, not lib/
```xml
<BuildOutputTargetFolder>analyzers/dotnet/cs/</BuildOutputTargetFolder>
```